Assets/XML/Text/Victory.xml

```diff
@@ -1539,19 +1539,19 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_UHV_JAP1_EPIC</Tag>
```

Comment:
I think the Epic and Marathon entries can be removed entirely now that the turns are displayed programmatically.

Reply:
But you still have different culture values you need to achieve on each speed?

Reply:
Oh right, forget about this.